resources/sdk/purecloudpython/templates/requirements.txt

```diff
@@ -0,0 +1,6 @@
+certifi >= 14.05.14
```

Comment:
You'll have to update PythonClientCodegen.java in swagger codegen and change requirements.mustache to requirements.txt

Reply:
PR is up - https://github.com/purecloudlabs/swagger-codegen/pull/49